src/modules/database/database.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable no-console */
+import mongoose from 'mongoose';
+
+class Database {
+  private static instace: Database | null = null;
+
+  private mongodb = mongoose;
+
+  private uri = 'mongodb://127.0.0.1:27017/local';
+
+  private constructor() {}
+
+  public static getInstance(): Database {
+    if (Database.instace === null) Database.instace = new Database();
+
+    return Database.instace;
+  }
+
+  public async connect(config =
+  { useNewUrlParser: true, useUnifiedTopology: true }): Promise<void> {
+    await this.mongodb.connect(this.uri, config,
```

Comment:
How about changing this to `try catch` syntax ?

Reply:
Thanks for the advise.